src/vorta/views/repo_tab.py

```diff
@@ -124,6 +129,12 @@ def init_repo_stats(self):
             self.sshComboBox.setEnabled(ssh_enabled)
             self.sshKeyToClipboardButton.setEnabled(ssh_enabled)
 
+            # Disable the change passphrase button if encryption type is not repokey
+            if repo.encryption in ['repokey', 'repokey-blake2']:
+                self.changePassbutton.setEnabled(True)
+            else:
+                self.changePassbutton.setEnabled(False)
```

Comment:
Can you add a reason to the tooltip when disabled? E.g. `some tooltip (no repokey encryption)`